main.js

```diff
@@ -0,0 +1,103 @@
+const start = () => {
+  console.log(``);
+  console.log(nameExercise);
+  console.log(``);
+};
+// ======================================
+let nameExercise = `Prototype inheritance`;
+start();
+
+ function University  (universityName, dean){
+  this.universityName = universityName;
+  this.dean = dean;
+};
+
+ function Faculty  (universityName, dean,facultyName){
+ University.call(this,universityName, dean);
+ this.facultyName = facultyName;
+ this.groups = {};
+};
+
+Faculty.prototype = Object.create(University.prototype);
+Faculty.prototype.constructor = Faculty;
+
+Faculty.prototype.enlistStudent = function(groupName,studentName){
+  if(!this.groups[groupName]){
+    this.groups[groupName] = [];
+  }
+  if ( this.groups[groupName].length < 12){
+      this.groups[groupName].push(studentName);
+  }else{
+    console.log(`Full groups.`);
+  }
+}
+const university = new University(`Polytechnic`,`Kohen`);
+const faculty = new Faculty(`Polytechnic`,`Kohen`,`Matematic`);
```

Comment:
great work but you misunderstood the task
make it as an object and use `Object.create`